### PR TITLE
Single Add ( User to key in ZRR Manager )

### PR DESCRIPTION
**Pull request by yswei93**
1

### CODE REVIEW

File: modal/create_bingx_account_modal.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import discord
+import message as ms
+from discord.interactions import Interaction
+
+TITLE = "Submit Your Bing X Account Details"
+
+
+class CreateBingXAccountModal(discord.ui.Modal, title=TITLE):
```
Comment (kingsleylow0327) [inline modal/create_bingx_account_modal.py]: rename this to `uuidSubmission`

File: view/master_view.py [hunk 2]
```diff
@@ -18,6 +19,10 @@ def __init__(self, dbcon, user_account_list, license_list = None):
     #                                                                    self.user_account_list,
     #                                                                    "trader"))
 
+    @discord.ui.button(label="Create Account", style=discord.ButtonStyle.grey)
```
Comment (kingsleylow0327) [inline view/master_view.py]: rename this to `UUID Submission`

File: view/master_view.py [hunk 3]
```diff
@@ -18,6 +19,10 @@ def __init__(self, dbcon, user_account_list, license_list = None):
...
+    async def createAccountButton(self, interaction: discord.Interaction, button: discord.ui.Button):
```
Comment (kingsleylow0327) [inline view/master_view.py]: Rename this function too

File: modal/create_bingx_account_modal.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        self.user_account_name = selected_account
+        super().__init__(title=TITLE, timeout=120)
+
+    uuid = discord.ui.TextInput(label="Bing X UUID", placeholder="Bing X UUID", style=discord.TextStyle.short)
+
+    async def on_submit(self, interaction: Interaction):
+        uuid = self.uuid.value
+        discord_id = interaction.user.id
+        if self.dbcon.check_discord_id_exist_from_trade_volume_table(discord_id):
+            await interaction.response.send_message(ms.REGISTERED_DISCORD_ID, ephemeral=True)
```
Comment (kingsleylow0327) [inline modal/create_bingx_account_modal.py]: Can we change this to:
If account existed, update
Else insert

**Review by kingsleylow0327 — COMMENTED** [review]
Please take a look

File: modal/create_bingx_account_modal.py (hunk 1 above)
Comment (yswei93) [reply modal/create_bingx_account_modal.py:8]: ok

File: view/master_view.py (hunk 2 above)
Comment (yswei93) [reply view/master_view.py:22]: ok

File: view/master_view.py (hunk 3 above)
Comment (yswei93) [reply view/master_view.py:23]: ok

File: modal/create_bingx_account_modal.py (hunk 4 above)
Comment (yswei93) [reply modal/create_bingx_account_modal.py]: fixed

File: view/master_view.py (hunk 3 above)
Comment (yswei93) [reply view/master_view.py]: fixed

File: view/master_view.py (hunk 2 above)
Comment (yswei93) [reply view/master_view.py]: fixed

File: modal/create_bingx_account_modal.py (hunk 1 above)
Comment (yswei93) [reply modal/create_bingx_account_modal.py]: fixed

**Review by kingsleylow0327 — APPROVED** [review]
LGTM

**Pull request merged**